Formula/fcgene.rb

```diff
@@ -0,0 +1,23 @@
+class Fcgene < Formula
+  desc "Format-converting tool for genotype data"
+  homepage "https://sourceforge.net/projects/fcgene/"
```

Comment:
The homepage seems to actually be https://www.imise.uni-leipzig.de/en/Groups/GenStat/Tools/index.jsp